converters/fromZigbee.js

```diff
@@ -6605,6 +6605,16 @@ const converters = {
             }
         },
     },
+    aqara_E1_switch_mode_switch: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty(0x0004)) {
```

Comment:
Please merge this converter with https://github.com/Koenkk/zigbee-herdsman-converters/blob/32671a8c402d7efc7c4cfcf84cea28f7c59bd9b9/converters/fromZigbee.js#L4488